planet/clients/data.py

```diff
@@ -185,11 +188,99 @@ async def create_search(self,
         response = await self._do_request(request)
         return response.json()
 
+    async def update_search(self, search_id: str,
+                            name: str,
+                            item_types: typing.List[str],
+                            search_filter: dict,
+                            enable_email: bool = False) -> dict:
+        """Update an existing saved search.
+
+        Parameters:
+            search_id: Saved search identifier.
+            name: The name of the saved search.
+            item_types: The item types to include in the search.
+            search_filter: Structured search criteria.
+            enable_email: Send a daily email when new results are added.
+
+        Returns:
+            Description of the saved search.
+        """
+        raise NotImplementedError
+
+    async def list_searches(
+            self,
+            sort: str = 'created desc',
+            search_type: str = 'any') -> typing.AsyncIterator[dict]:
+        """List all saved searches available to the authenticated user.
+
+        NOTE: the term 'saved' is overloaded here. We want to list saved
+        searches that are 'quick' or list saved searches that are 'saved'? Do
+        we want to introduce a new term, 'stored' that encompasses 'saved' and
+        'quick' searches?
```

Comment:
I just want to highlight this and raise it for consideration of review of the Data API docs (my source for most of the docstrings).